central/cve/converter/utils/convert_utils_v2.go

```diff
@@ -0,0 +1,108 @@
+package utils
+
+import (
+	"strconv"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/cve"
+)
+
+// ImageCVEV2ToEmbeddedVulnerability coverts a `*storage.ImageCVEV2` to an `*storage.EmbeddedVulnerability`.
+func ImageCVEV2ToEmbeddedVulnerability(vuln *storage.ImageCVEV2) *storage.EmbeddedVulnerability {
+	scoreVersion := storage.EmbeddedVulnerability_V2
+	if vuln.GetCveBaseInfo().GetCvssV3() != nil {
+		scoreVersion = storage.EmbeddedVulnerability_V3
+	}
+
+	ret := &storage.EmbeddedVulnerability{
+		Cve:                   vuln.GetCveBaseInfo().GetCve(),
+		Cvss:                  vuln.GetCvss(),
+		Summary:               vuln.GetCveBaseInfo().GetSummary(),
+		Link:                  vuln.GetCveBaseInfo().GetLink(),
+		ScoreVersion:          scoreVersion,
+		CvssV2:                vuln.GetCveBaseInfo().GetCvssV2(),
+		CvssV3:                vuln.GetCveBaseInfo().GetCvssV3(),
+		PublishedOn:           vuln.GetCveBaseInfo().GetPublishedOn(),
+		LastModified:          vuln.GetCveBaseInfo().GetLastModified(),
+		FirstSystemOccurrence: vuln.GetCveBaseInfo().GetCreatedAt(),
+		Severity:              vuln.GetSeverity(),
+		CvssMetrics:           vuln.GetCveBaseInfo().GetCvssMetrics(),
+		NvdCvss:               vuln.GetNvdcvss(),
+		Epss:                  vuln.GetCveBaseInfo().GetEpss(),
+		FirstImageOccurrence:  vuln.GetFirstImageOccurrence(),
+		VulnerabilityType:     storage.EmbeddedVulnerability_IMAGE_VULNERABILITY,
+		VulnerabilityTypes:    []storage.EmbeddedVulnerability_VulnerabilityType{storage.EmbeddedVulnerability_IMAGE_VULNERABILITY},
+		State:                 vuln.GetState(),
+	}
+
+	if vuln.IsFixable {
```

Comment:
nit: `vuln.GetIsFixable()`

Reply:
I will defer this one.  I'll get it in with one of the many subsequent PRs.